dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -60,8 +60,8 @@ C3.1,"ObligationsUndeliveredOrdersUnpaidTotal (FYB or CPE) = USSGL(4801 + 4881).
 C3.2,"ObligationsUndeliveredOrdersUnpaidTotal (FYB or CPE) = USSGL(4801 + 4881). This applies to the award level.","ObligationsUndeliveredOrdersUnpaidTotalFYB = USSGL4801. This applies to the award level.",n,award_financial,warning,,c3_award_financial_2
 C4.1,"ObligationsDeliveredOrdersUnpaidTotal (FYB or CPE) = USSGL(4901 + 4981). This applies to the award level.","ObligationsDeliveredOrdersUnpaidTotalCPE = USSGL(4901 + 4981). This applies to the award level.",n,award_financial,warning,,c4_award_financial_1
 C4.2,"ObligationsDeliveredOrdersUnpaidTotal (FYB or CPE) = USSGL(4901 + 4981). This applies to the award level.","ObligationsDeliveredOrdersUnpaidTotalFYB = USSGL4901. This applies to the award level.",n,award_financial,warning,,c4_award_financial_2
-C5.1,"GrossOutlayAmountByAward (FYB or CPE, File C) = GrossOutlaysUndeliveredOrdersPrepaidTotal (FYB or CPE, File C) + GrossOutlaysDeliveredOrdersPaidTotal (FYB or CPE, File C)",GrossOutlayAmountByAwardCPE = GrossOutlaysUndeliveredOrdersPrepaidTotalCPE + GrossOutlaysDeliveredOrdersPaidTotalCPE,n,award_financial,warning,,c5_award_financial_1
-C5.2,"GrossOutlayAmountByAward (FYB or CPE, File C) = GrossOutlaysUndeliveredOrdersPrepaidTotal (FYB or CPE, File C) + GrossOutlaysDeliveredOrdersPaidTotal (FYB or CPE, File C)",GrossOutlayAmountByAwardFYB = GrossOutlaysUndeliveredOrdersPrepaidTotalFYB + GrossOutlaysDeliveredOrdersPaidTotalFYB,n,award_financial,warning,,c5_award_financial_2
+C5.1,"GrossOutlayAmountByAward (FYB or CPE, File C) = GrossOutlaysUndeliveredOrdersPrepaidTotal (FYB or CPE, File C) + GrossOutlaysDeliveredOrdersPaidTotal (FYB or CPE, File C)","GrossOutlayAmountByAwardCPE = GrossOutlaysUndeliveredOrdersPrepaidTotalCPE + GrossOutlaysDeliveredOrdersPaidTotalCPE",n,award_financial,warning,,c5_award_financial_1

```

Comment:
Instead of `FYB or CPE` shouldn't it be `CPE - FYB`? or something describing the subtraction of values.

Reply:
The first set of text isn't the text that anyone sees. They see the "rule error message" portion. This is just for the DB, but you're right that the second half should have the FYB mentioned